integration-tests/__tests__/each.test.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const path = require('path');
+const SkipOnWindows = require('../../scripts/SkipOnWindows');
+const runJest = require('../runJest');
+const {extractSummary} = require('../Utils');
+const dir = path.resolve(__dirname, '../each');
+
+SkipOnWindows.suite();
```

Comment:
it would be great if this was not needed. if it is, no biggie

Reply:
I can't see why it wouldn't work on windows so I'll remove it and see what happens :)